osu.Game.Rulesets.Rush/Objects/Drawables/DrawableSawblade.cs

```diff
@@ -68,13 +68,19 @@ protected override void CheckForResult(bool userTriggered, double timeOffset)
             if (userTriggered || timeOffset < 0 || AllJudged)
                 return;
 
-            if (HitObject.HitWindows.ResultFor(timeOffset) != HitResult.None)
-                return;
+            switch (HitObject.HitWindows.ResultFor(timeOffset))
+            {
+                case HitResult.None:
+                    ApplyResult(r => r.Type = HitResult.Perfect);
```

Comment:
got me very confused on first glance, but now I understand what this means (timeOffset is beyond all result windows)
```suggestion
                    // Passed the sawblade, can apply perfect result now.
                    ApplyResult(r => r.Type = HitResult.Perfect);
```

Reply:
I guess.